audbackend/core/backend.py

```diff
@@ -213,7 +213,9 @@ def join(
             path joined by :attr:`Backend.sep`
 
         """
-        return self.sep.join([path] + [p for p in paths])
+        paths = [path] + [p for p in paths]
+        paths = [path for path in paths if path]
```

Comment:
Could you add a comment above this line stating that it removes `''` path entries. Because otherwise it might be hard to get later on.

Reply:
done